src/banach/pagination.clj

```diff
@@ -0,0 +1,23 @@
+(ns banach.pagination
+  (:require [manifold.stream :as ms]
+            [manifold.deferred :as md]))
+
+(defn paginated->stream
+  "Given a resource with pagination semantics, turns it into a single
```

Comment:
What about "Given a resource with pagination semantics, _returns_ a single stream that ..."